examples/tracing/tracing.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/runtime;
+import ballerina/observe;
+
+// Make sure you start the service with --observe, or tracing enabled.
+service<http:Service> hello bind { port: 9090 } {
+
+    // Invoke all resources with arguments of server connector and request.
+    sayHello(endpoint caller, http:Request req) {
+        http:Response res = new;
+
+        //Start a child span attaching to the system span generated.
+        int spanId = check observe:startSpan("MyFirstLogicSpan");
+
+        //Start a new Root span without attaching to the system span.
+        int rootParentSpanId = observe:startRootSpan("MyRootParentSpan");
+        // Some actual logic will go here, and for example we have introduced some delay with sleep.
+        runtime:sleep(1000);
+        //Start a new child span for the span `MyRootParentSpan`.
+        int childSpanId = check observe:startSpan("MyRootChildSpan", parentSpanId = rootParentSpanId);
+        // Some actual logic will go here, and for example we have introduced some delay with sleep.
+        runtime:sleep(1000);
+        //Finish `MyRootChildSpan` span.
+        _ = observe:finishSpan(childSpanId);
+        // Some actual logic will go here, and for example we have introduced some delay with sleep.
+        runtime:sleep(1000);
+        //Finish `MyRootParentSpan` span.
+        _ = observe:finishSpan(rootParentSpanId);
+
+        // Some actual logic will go here, and for example we have introduced some delay with sleep.
+        runtime:sleep(1000);
+
+        //Finish the created child span `MyFirstLogicSpan` which was attached to the system trace.
```

Comment:
Add a comma before 'which'. Normally we have a comma before 'which' in this context or simply use 'that' instead. Apply elsewhere and in other PRs where possible.